extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -55,32 +59,48 @@ function jetpack_mailchimp_block_load_assets( $attr ) {
 	}
 	$classes = implode( $classes, ' ' );
 
+	$button_styles = array();
```

Comment:
Noting that it's possible to pass any other CSS via these attributes (even as contributor level user — I'm not sure if they normally would be allowed `style` attribute by `wp_kses`) since we don't validate for HEX colors.

So e.g. funky border:
<img width="234" alt="image" src="https://user-images.githubusercontent.com/87168/53397371-aa41a680-39af-11e9-8a45-c50324df360a.png">

I don't think needs changes for now.

Reply:
> since we don't validate for HEX colors

Maybe `sanitize_hex_color` could be useful here?

Reply:
Nice! TIL

https://developer.wordpress.org/reference/functions/sanitize_hex_color/

Reply:
done in d72fd2f3ecb0b53d9c55e30bb48b3c35ae5604e1